Sources/Sentry/SentrySystemEventBreadcrumbs.m

```diff
@@ -278,6 +282,27 @@ - (void)updateStoredTimezone
         storeTimezoneOffset:SentryDependencyContainer.sharedInstance.dateProvider.timezoneOffset];
 }
 
+- (void)initSignificantTimeChangeObserver
+{
+
+    [self.notificationCenterWrapper addObserver:self
+                                       selector:@selector(significantTimeChangeTriggered:)
+                                           name:UIApplicationSignificantTimeChangeNotification
+                                         object:nil];
+}
+
+- (void)significantTimeChangeTriggered:(NSNotification *)notification
```

Comment:
`l`: I had to look up if the notification contains any other relevant information we should add to the breadcrumb. It does not. Added docs so other don't have to search too.

```suggestion
/**
 * The system posts this notification when, for example, there’s a change to a new day (midnight), a carrier time update, or a change to, or from, daylight savings time. 
 * The notification doesn’t contain a user info dictionary.
 *
 * @see https://developer.apple.com/documentation/uikit/uiapplication/significanttimechangenotification#Discussion
 */
- (void)significantTimeChangeTriggered:(NSNotification *)notification
```